app/krs.js

```diff
@@ -186,5 +186,14 @@ exports.provisionKey = co(function *(req) {
     yield notifyEndpoint(key, 'created');
   }
 
-  return key;
```

Comment:
This return value gets sent directly to the WWW platform. Previously, this was returning a mongo object, which caused error when WWW tried to read the info. The fix is to instead return the regular json object, below, with only the necessary fields included.